odoo/tests/loader.py

```diff
@@ -44,13 +44,22 @@ def make_suite(module_names, position='at_install'):
     """
     config_tags = TagsSelector(tools.config['test_tags'])
     position_tag = TagsSelector(position)
-    tests = (
-        t
-        for module_name in module_names
-        for m in get_test_modules(module_name)
-        for t in unwrap_suite(unittest.TestLoader().loadTestsFromModule(m))
-        if position_tag.check(t) and config_tags.check(t)
-    )
+    loader = unittest.TestLoader()
+
+
+    test_cases = []
+    for module_name in module_names:
+        for m in get_test_modules(module_name):
+            for t in unwrap_suite(loader.loadTestsFromModule(m)):
+                if not isinstance(t, CrossModule):
+                    test_cases.append(t)
+        for c in CrossModule.registry:
```

Comment:
What if `c.test_module` is not in the modules loaded by the registry?

Reply:
I think they are in the registry by definition since only the loaded modules will be passed to make_suite 

```python
                module_names = (registry.updated_modules if update_module else
                                sorted(registry._init_modules))
                _logger.info("Starting post tests")
                tests_before = registry._assertion_report.testsRun
                post_install_suite = loader.make_suite(module_names, 'post_install')
```